internal/host/hostcommands/logs_cmd.go

```diff
@@ -83,10 +83,15 @@ func createUploadLogsCmd(host *models.Host, baseURL, agentImage, mastersIPs stri
 		"BOOTSTRAP":              strconv.FormatBool(host.Bootstrap),
 		"INSTALLER_GATHER":       strconv.FormatBool(withInstallerGatherLogging),
 		"MASTERS_IPS":            mastersIPs,
-		"CACERTPATH":             common.HostCACertPath,
 	}
+
+	if i.instructionConfig.ServiceCACertPath != "" {
```

Comment:
Isn't the `{{if .CACERTPATH}}` inside the template enough? Why do we need this `if` as well?

Reply:
Oh nevermind ignore me